rein/lib/rating.py

```diff
@@ -115,18 +115,17 @@ def get_average_user_rating(log, url, user, rein, msin):
     """Gets the average rating a user (identified by his msin) has received
     along with the number of ratings he has received"""
 
-    sel_url = "{0}query?owner={1}&delegate={2}&query=get_user_ratings&testnet={3}&msin={4}"
```

Comment:
If we don't query for ratings from the server here, where and when do we still obtain ratings and put them in our local client database?